### PR TITLE
Update whoami.c

### PR DESCRIPTION
**Pull request by Cloudmichael**

### CODE REVIEW

File: whoami.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 
 int main() {
+	printf("1023026");
```
Comment (jrjang2) [inline whoami.c]: Leave an empty line before ```return 0;```.

**Comment by jrjang2** [comment]
Merge your two commits to one before doing pull request.

Tips:
1. Try to use ```git rebase```.
2. Use ```git reset --mixed``` twice, and commit again.

**Comment by Cloudmichael** [comment]
這樣可以了嗎

**Comment by jrjang2** [comment]
Good. But there is a conflict now. Please solve this issue before doing pull request. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (git pull --rebase) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by Cloudmichael** [comment]
好了!!

**Comment by jrjang2** [comment]
There are many redundant commits, such as "Merge ...". The reason is that you never doing rebase.

Please follow the below instructions.

1. Fork again on github.
2. ```git pull --rebase```.
3. pull request.

**Comment by Cloudmichael** [comment]
ok!

**Comment by Cloudmichael** [comment]
ok!!

**Comment by jrjang2** [comment]
Great. But there is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by Cloudmichael** [comment]
.

**Comment by jrjang2** [comment]
Good!

**Pull request merged**